src/schema/hotel-price-crawling.json

```diff
@@ -129,6 +129,46 @@
                 },
                 "canBeShared": {
                     "type": "boolean"
+                },
+                "bed": {
+                    "type": "array",
+                    "description": "Types of the beds.",
+                    "minItems": 0,
+                    "items": {
+                        "type": "string",
+                        "enum": [
+                            "single",
+                            "double",
+                            "semi-double",
+                            "queen",
+                            "king",
+                            "twin",
+                            "3",
+                            "4",
+                            "more than 4"
+                        ]
+                    }
+                },
+                "style": {
+                    "type": "string",
+                    "description": "Style of room for japanese hotels.",
+                    "enum": [
+                        "japanese",
+                        "western"
+                    ]
+                },
+                "features": {
+                    "type": "array",
+                    "description": "Room characteristics",
+                    "minItems": 0,
+                    "items": {
+                        "type": "string",
+                        "enum": [
+                            "smoking",
+                            "non-smoking",
+                            "openAirBath"
```

Comment:
```suggestion
                            "open-air-bath"
```

Reply:
```js
features: ['smoking', 'open-air-bath'], // 👌 
```
```js
features: ['smoking', 'non-smoking'], // ❓ 
```

Reply:
Won't it be better to have it as booleans?
```js
smoking: boolean,
openAirBath: boolean
```
This way, it's impossible to have contradictory data.